app/assets/javascripts/discourse/app/controllers/flag.js

```diff
@@ -181,6 +181,17 @@ export default Controller.extend(ModalFunctionality, {
 
   submitDisabled: not("submitEnabled"),
 
+  @discourseComputed("notifyModeratorsFlag")
+  cantFlagForReview(notifyModeratorsFlag) {
+    return !notifyModeratorsFlag;
```

Comment:
We can just use [not](https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/not) function here.